core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.controller.ts

```diff
@@ -196,6 +196,13 @@ angular.module('oppia').controller(
 
         [$scope.activeSuggestionId, $scope.activeContribution] = (
           remainingContributions.pop());
+        // Close modal instance if activeContribution.details is null.
```

Comment:
I think "activeContribution.details is null" is redundant from the code—we should describe what we're checking for at a higher level, i.e. "if the suggestion's corresponding opportunity is deleted"

Reply:
done